themes/src/main/resources/theme/keycloak/common/resources/node_modules/@patternfly/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -0,0 +1,309 @@
+// This file is for initial variable values.
+// All variables should have a valid css value,
+// and use !default so they can be overwritten.
+
+// `$pf-global--concept--PropiertyCamelCase--modifier--state`
+
+// Patternfly options
+//
+// Quickly modify global styling by enabling or disabling optional features.
+$pf-global--enable-reset: false !default;
```

Comment:
> Also, the possibility of simultaneous usage of PF3 and PF4 is enabled here:

Just a question. Does this mean this line was manually edited? How possible future update will be handled with manual changes here?

Reply:
@vmuzikar yes, you're correct, this line was edited manually according to the instructions mentioned here:
https://github.com/patternfly/patternfly/blame/master/src/site/pages/upgrade-guide.md#L13
Regarding the future update we will simply set the property back to the default value.

Reply:
> 
> 
> Thanks @Pepo48! Changes as such look good to me but at the same time I'm not sure if this is a valid change as a standalone. I understand that the ultimate goal is to migrate login theme from PF3 to PF4, correct? But what this PR does is it "just" adds a new dependency to the project without actually using it. So e.g. if there were a release in the mean time, it would include a lots of new files that would be unused. But I also understand that the point here is to make small steps in each PR. So I'll leave this up to @ssilvert whether this is ok or not.

@vmuzikar exactly, that was actually the main question - what approach to choose? Whether to create one big PR with the package upgrade and all the screens update or divide it to smaller steps... We've chose the second option and therefore none of the theme.properties files is changed in this PR.